3ds2/src/main/java/com/adyen/checkout/adyen3ds2/Adyen3DS2Delegate.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2022 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by oscars on 22/8/2022.
+ */
+
+package com.adyen.checkout.adyen3ds2
+
+import android.app.Activity
+import android.content.Intent
+import com.adyen.checkout.components.model.payments.response.Action
+import com.adyen.checkout.core.exception.CheckoutException
+import com.adyen.threeds2.customization.UiCustomization
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.flow.Flow
+import org.json.JSONObject
+
+interface Adyen3DS2Delegate {
+
+    val detailsFlow: Flow<JSONObject>
+
+    val exceptionFlow: Flow<CheckoutException>
+
+    val eventFlow: Flow<Adyen3DS2Event>
+
+    var uiCustomization: UiCustomization?
+
+    val gotDestroyedWhileChallenging: Boolean
```

Comment:
Wondering if this will even work, `gotDestroyedWhileChallenging` was a static field before because the whole component would be destroyed and recreated which now the value will be set to true but the delegate reference will be lost anyway so it will be `false` again with the new delegate.

I think we can just remove it completely, all it did was log a line and we already have a check for transaction availability in `challengeShopper` with an exception. WDYT?

Reply:
You're right, I think it's safe to remove